scss/_variables.scss

```diff
@@ -23,3 +23,5 @@ $code-color: #e83e8c !default;
 
 $lead-font-size: 1.75rem !default;
 $lead-font-weight: 400 !default;
+
+$enable-smooth-scroll: false;
```

Comment:
I think it would probably be fine to just turn it off entirely -- I don't think it really adds anything